.github/workflows/integration-test.yaml

```diff
@@ -28,13 +28,11 @@ jobs:
         run: echo "charm_path=$(find built/ -name '*.charm' -type f -print)" >> $GITHUB_OUTPUT
       
       - name: Setup operator environment
-        uses: charmed-kubernetes/actions-operator@main
-        with:
-          provider: microk8s
-          channel: 1.29-strict/stable
-          juju-channel: 3.4/stable
-          lxd-channel: 5.20/stable
-  
+        run: |
+          sudo snap install concierge --classic
```

Comment:
Is the intent to use concierge in all of our charm's CI? Does this change belong here? It seems independent from the `uv` change

Reply:
The switch to concierge was done because we need to install tox differently, and the action we used previously is hardcoded to install tox. There is no way to prevent it from installing it, or of forcing it to install it like we need.

The alternative is to keep the action, but remove tox afterwards and install it properly.

I prefer concierge in that case because it enables us to run the exact same thing locally.

Reply:
If concierge is ready for wide use (i.e. in all of Charm engineering repos), I'm fine with it.